status.py

```diff
@@ -47,6 +48,7 @@ def is_server_error(code):
     204: 'HTTP_204_NO_CONTENT',
     205: 'HTTP_205_RESET_CONTENT',
     206: 'HTTP_206_PARTIAL_CONTENT',
+    207: 'HTTP_207_MULTI_STATUS',
     226: 'HTTP_226_IM_USED',
```

Comment:
```suggestion
    226: 'HTTP_226_IM_USED',
```
and also 
425: 'HTTP_425_ TOO_EARLY',